shell/README.md

```diff
@@ -17,15 +17,17 @@ adding the following snippet:
 
 ### Zsh
 
+Load up the completion by placing the `exercism_completion.zsh` somewhere on
+your `$fpath` as `_exercism`. For example:
+
     mkdir -p ~/.config/exercism
-    mv ../shell/exercism_completion.zsh ~/.config/exercism/exercism_completion.zsh
+    mv ../shell/exercism_completion.zsh ~/.config/exercism/_exercism
```

Comment:
I do not like those living in `~/.config/exercism`, this is not a configuration file!

A more idiomatic way is to use `~/.zsh`, `~/.zsh/functions`, or `~/.zsh/completions`. Some folks might even use something like `~/.config/zsh/…`, but I haven't seen this very often in the dotfiles I studied so far.

Reply:
I don't really like it either, but I was trying to leave the change as minimal as possible.

Reply:
@QuLogic I would be totally fine with having a more intrusive change, if it makes it better. Basically the shell completion scripts have not had nearly as much attention as the other parts of the code, so if this is something that you have skill and knowledge to contribute to, that would be very welcome, indeed!

I have never used zsh, so I will defer to @NobbZ with respect to the discussion of what is right and wrong.

Reply:
If I were using zsh, I'd prefer a more XDG-compliant directory like `~/.local/share/zsh/functions`, but since I don't, I'll go with your suggestion of `~/.zsh/functions`.